reactioncommerce/reaction
# Use route permissions on the registry

**Resolves**
#2297 

**Testing**
- Create a plugin according to the documentation
- Add permissions in the plugins `register.js` according to the documentation
- Try to access the defined route for your plugin. If the user is meant to have the permission to view the plugin then it should display. 

**Read**
Look into this documentation on creating routes for plugins
https://docs.reactioncommerce.com/reaction-docs/master/plugin-routes-6
